packages/dcos-integration-test/extra/conftest.py

```diff
@@ -17,6 +19,39 @@ def pytest_configure(config):
     config.addinivalue_line('markers', 'last: run test after all not marked last')
 
 
+def pytest_cmdline_main(config):
+    user_outside_cluster = True
+    if os.path.exists('/opt/mesosphere/bin/dcos-shell'):
```

Comment:
I think, this "feature" should be commented or explained as why we choose to do this. Otherwise the rest of the changes look good to me.